pixel-point/pixelpoint-website
# chore: add parca case study

**Describe what changes this pull request brings**
This pull request adds Parca case study.

**Steps to test**

1. Open the [page](https://deploy-preview-65--pixelpoint-website.netlify.app/case-studies/)
2. Confirm that everything looks and works as expected

**Screenshots**
Parca case study
<img width="2048" alt="Screenshot 2022-05-12 at 17 26 58" src="https://user-images.githubusercontent.com/75138180/168074827-a7182219-f9e3-448e-a743-b94191240ddb.png">





**References**

Design at [Figma](https://www.figma.com/file/uRyIVXOk6qD7ufNv6JSQRf/Pixel-Point-Website-Design?node-id=3051%3A96335)
